retrospective-update/cloud_logger.py

```
@@ -7,12 +7,19 @@
 
-LOG_GROUP_NAME = os.getenv('AWS_LOG_GROUP_NAME')
-LOG_STREAM_NAME = os.getenv('AWS_LOG_STREAM_NAME')
-ACCESS_KEY_ID = os.getenv('AWS_ACCESS_KEY_ID')
-SECRET_ACCESS_KEY = os.getenv('AWS_SECRET_ACCESS_KEY')
-REGION = os.getenv('REGION_NAME')
-
+if False:
+    # Set up logging
+    logging.basicConfig(
+        level=logging.INFO,
+        filename=f'log.log',
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)